cypress/e2e/alerts.cy.js

```diff
@@ -1,25 +1,57 @@
 describe('Cypress application', () => {
-  before(() => {
-
+  beforeEach(() => {
+    cy.visit('https://demoqa.com/alerts')
   });
 
   it('should have the ability to assert automatically resolved alerts', () => {
+    cy.get('#alertButton')
+      .click();
 
+    cy.on('window:alert', (str) => {
+      expect(str).to.contains('You clicked a button');
+    })
   });
 
   it('should have the ability to assert scheduled allert', () => {
+    cy.get('#timerAlertButton')
+      .click();
+
+    cy.wait(5000);
 
+    cy.on('window:alert', (str) => {
+      expect(str).to.contains('This alert appeared after 5 seconds');
+    });
   });
 
   it('should autimatically resolve alerts', () => {
+    cy.get('#confirmButton')
+      .click();
+
+    cy.on('window:confirm', (str) => {
+      expect(str).to.contains('Do you confirm action?');
+    });
 
+    cy.get('#confirmResult')
+      .should('contain', 'You selected Ok');
   });
 
   it('should have the ability to Cancel alerts', () => {
+    cy.get('#confirmButton')
+      .click();
 
-  });
+    cy.on('window:confirm', cy.stub().returns(false));
 
-  it('should have the ability to enter text to alert', () => {
+    cy.get('#confirmResult')
+    .should('contain', 'You selected Cancel');
+  });
 
+  it.only('should have the ability to enter text to alert', () => {
```

Comment:
Remove 'only' to make sure that all tests will be run

Reply:
Forgot about that) Done